frontend/src/views/registration_page/registrationPage.js

```diff
@@ -107,22 +108,22 @@ class RegistrationPage extends React.Component {
 
 
                                 <input type="submit" className="btn" value="Sign up" />
-                                <p style={{color: "black"}}>Already a User? <a href="#">Sign in</a></p>
+                                <p style={{color: "black"}}>Already a User? Sign in</p>
```

Comment:
You can add like react-router Link  rapped around the Sign in (Link to="/login")after the **Already a User?**